pontoon/sync/tasks.py

```diff
@@ -127,6 +128,12 @@ def sync_project(
         full_scan=force
     )
 
+    new_entities = source_changes.get('new_entities')
+
+    if db_project.pretranslation_enabled and new_entities:
+        new_entities = list(set(new_entities))
+        pretranslate(db_project, entities=new_entities)
```

Comment:
We sometimes "migrate" translations, e.g. from one format to another (recently that happens often with Firefox, where we're replacing legacy file formats with Fluent). In practice that's done via a script and committed to VCS. On next sync, we then create new entities as part of `sync_sources()` and new translations as part of `sync_translations()`.

The logic above will pretranslate such entities, even though it shouldn't (because we'll create translations from VCS immediately after within `sync_translations()`. So we should delay execution of `pretranslate()` once all translations are created.